### ISSUE TITLE
Error: Unresolved node modules: react-native

### ISSUE THREAD

**Issue by ThaJay** [PaulLeCam/react-native-electron#4]
Perhaps I'm in the wrong place here, excuse me if I am. This may also be more of a question than an issue.
I also posted a stackoverflow question: https://stackoverflow.com/questions/50620640/electron-builder-tries-to-include-react-native-while-i-have-an-alias-set-in-webp

When I try `yarn dist` as in 
```
"compile": "electron-webpack", 
"dist": "yarn compile && electron-builder",
```
it errors out with
```
Error: Unresolved node modules: react-native
    at /media/dj/dev/scan-app-desktop/node_modules/electron-builder-lib/src/util/packageDependencies.ts:108:17
From previous event:
    at Collector.resolveUnresolvedHoisted (/media/dj/dev/scan-app-desktop/node_modules/electron-builder-lib/src/util/packageDependencies.ts:164:10)
    at /media/dj/dev/scan-app-desktop/node_modules/electron-builder-lib/src/util/packageDependencies.ts:82:18
    at Generator.next (<anonymous>)
    at runCallback (timers.js:763:18)
    at tryOnImmediate (timers.js:734:5)
    at processImmediate (timers.js:716:5)
From previous event:
    at Collector.collect (/media/dj/dev/scan-app-desktop/node_modules/electron-builder-lib/src/util/packageDependencies.ts:84:6)
    at /media/dj/dev/scan-app-desktop/node_modules/electron-builder-lib/out/util/packageDependencies.js:123:47
    at Generator.next (<anonymous>)
(...)
```

in my package.json I have the electronWebpack config:
```
  "electronWebpack": {
    "renderer": {
      "webpackConfig": "webpack.renderer.additions.js"
    }
  },
```
and in webpack.renderer.additions.js:
```
const path = require('path')

const resolve = {
  alias: {
    'react-native': path.join(__dirname, 'node_modules/react-native-electron')
  }
}

module.exports = {resolve}
```

**Comment by PaulLeCam** [comment]
Sorry I'm not familiar with how Electron webpack works, but you might want to try using the alias name directly and let webpack do the resolution itself:

```js
alias: {
  'react-native': 'react-native-electron'
}
```

**Comment by ThaJay** [comment]
Turns out this was due to `electron-builder install-app-deps`, so this was not related to `react-native-electron` but `electron-webpack`. It checks dependencies in package.json and tries to include them with their subdependencies. So if I move the packages that are bundled by webpack to devDependencies it builds.

Thank you for trying to help and sorry for posting in the wrong place, I got a bit confused as to who does what exactly.
